qiskit_nature/drivers/second_quantization/__init__.py

```diff
@@ -25,8 +25,7 @@
 already using the underlying chemistry program or library will be familiar with. The driver is
 used to compute some intermediate data, which later will be used to form the input to an
 algorithm.  Such intermediate data, is populated into a
-:class:`~qiskit_nature.drivers.second_quantization.QMolecule` object and includes the following for
-example:
+:class:`~qiskit_nature.drivers.QMolecule` object and includes the following for example:
```

Comment:
I am under the impression this is deprecated and going away and the drivers here emit some grouped property result now. This text I think ought to be updated in that regard etc.